### DISCUSSION TITLE
GLDF freelancer wanted

### DISCUSSION THREAD

**Discussion by dsacher16** [globallightingdata/gldf#27] [Q&A]
Hi, we are a lighting manufacturer looking for a freelancer who can create GLDF files for our lighting designers. We can provide LDT/IES files and 3D CAD files of the mechanical models (such as STEP, IGES, Parasolid) and we are looking for someone who can merge those data in consolidated GLDF files. Thanks Dominic

**Comment by holg** [comment]
Hi there,
well it depends on the input data and the amount.
So it would be known to make the code for it or to do it by hand.
Some basic data could be taken from the Eulumdat file, but this will not be sufficient.
For the 3d files it also depends, if you want to have the new .l3d file format as well...

**Comment by dsacher16** [comment]
Hi,

thanks for the quick answer, fairly simple 5 different length of linear
office lighting, yes .l3d format as well.

Input data for lighting : LDT or IES
mechanical data : STEP, IGS or Parasolid

with: 40mm
height : 60mm
length: 2100mm, 3000mm, 3500mm, 4300mm, 6200mm

kindly let me know the cost and time schedule, thanks a lot

mit freundlichen Grüßen / with best regards
Dominic Sacher
+49-151-24154096

Am 27.09.2024 um 11:21 schrieb Holger Trahe ***@***.***>:

﻿

Hi there,
well it depends on the input data and the amount.
So it would be known to make the code for it or to do it by hand.
For the 3d files it also depends, if you want to have the new .l3d file format as well...
